lua/neo-tree/sources/common/container.lua

```diff
@@ -270,8 +270,10 @@ local merge_content = function(context)
   vim.list_extend(result, left)
   vim.list_extend(result, right)
   context.merged_content = result
-  log.trace("wanted width: ", wanted_width, " actual width: ", context.container_width)
-  context.wanted_width = math.max(wanted_width, context.wanted_width)
+  if wanted_width > context.container_width then
```

Comment:
I'm not sure why this change is in there, but I noticed that autowidth is broken in this branch, which also means that the position="current" trees are squashed.